docs/tutorial/rdd.md

```diff
@@ -114,7 +114,7 @@ Use the following code to create a SpatialRDD
 ```Scala
 val inputLocation = "/Download/checkin.csv"
 val wktColumn = 0 // The WKT string starts from Column 0
-val allowTopologyInvalidGeometris = true // Optional
+val allowTopologyInvalidGeometries = true // Optional
 val skipSyntaxInvalidGeometries = false // Optional
 val spatialRDD = WktReader.readToGeometryRDD(sparkSession.sparkContext, inputLocation, wktColumn, allowTopologyInvalidGeometris, skipSyntaxInvalidGeometries)
```

Comment:
Thanks for your contribution! Can you change the typo here which is relevant to the one you fixed? allowTopologyInvalidGeometries in WktReader.readToGeometryRDD

Reply:
No problem! Just made the change you requested.